Form/Type/BooleanType.php

```diff
@@ -55,12 +56,18 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
+        $choices =  array(
+            self::TYPE_YES  => 'label_type_yes',
+            self::TYPE_NO   => 'label_type_no',
+        );
+
+        if (version_compare(Kernel::VERSION, '3.0', '>=')) {
```

Comment:
you could use httpkernel in version 3.0 but a lower form component....

the best thing to check this is to check for an existing/nonexisting class